apps/google-analytics-4/frontend/src/hooks/useSidebarSlug/useSidebarSlug.tsx

```diff
@@ -16,11 +18,26 @@ export const useSidebarSlug = (slugFieldInfo: ContentTypeValue) => {
     setIsPublished(Boolean(sys.publishedAt));
   };
 
+  useEffect(() => {
+    const timeout = setTimeout(() => setDebouncedSlugFieldValue(slugFieldValue), 700);
```

Comment:
Minor suggestion to pull out the value here (700) as a constant with a meaningful name.

Also, just curious why 700? Seems fairly reasonable, but we might want to try for lower (in general slug fields almost never change, my thinking is we'd probably want to tune this to a fairly low number)

Reply:
yes I was playing around with the delay here and 500 - 700 felt like a sweet spot of not allowing too many network calls while not making the user wait for the changes to reflect. Happy to pull out into a constant, and will lower down to 500!